src/main/java/frc/robot/commands/AutoSetShooter.java

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.RobotContainer;
+import frc.robot.Constants.AprilTag.ID;
+import frc.robot.subsystems.Limelight;
+import frc.robot.subsystems.ShooterFlywheel.SpinType;
+
+/**
+ * Calculate shooter angle and RPM using the robot's distance from the speaker.
+ * Assumes odometry is accurate and projectile flies in a straight line at calculated RPM.
+ * Works the best within alliance area.
+ */
+public class AutoSetShooter extends Command {
+  // Units in meters, positive = up or forward
+  public static final double speakerHeight = 2.18;
```

Comment:
What is this referring to? The lowest part of the speaker opening is 1.98 m from carpet, and the highest is 2.10 m from carpet.

Reply:
Fixed